pandas/core/internals/construction.py

```diff
@@ -550,7 +545,7 @@ def _list_of_dict_to_arrays(data, columns, coerce_float=False, dtype=None):
     """
     if columns is None:
         gen = (list(x.keys()) for x in data)
-        types = (dict, OrderedDict) if PY36 else OrderedDict
+        types = (dict, OrderedDict)
```

Comment:
maybe update docstring and inline types

Reply:
I think its also worth inlining types now that types is a less complex expression and only used once.